third_party/2and3/maxminddb/__init__.pyi

```diff
@@ -0,0 +1,9 @@
+# Stubs for maxminddb (Python 2 and 3)
```

Comment:
Let's just remove these comments; they're not useful.

Reply:
Done.